src/modules/curations/collectionCuration/utils.ts

```diff
@@ -15,9 +15,10 @@ export const getCollectionCurationState = (collection: Collection, curation: Col
       return CurationStatus.DISABLED
     } else if (curation && curation.status === 'rejected') {
       return CurationStatus.REJECTED
-    } else if (curation && curation.status === 'pending' && curation.assignee) {
-      return CurationStatus.UNDER_REVIEW
     }
   }
+  if (curation && curation.status === 'pending' && curation.assignee) {
```

Comment:
some collections that have `isApproved` as `true` (because they are in the catalyst) but they have a curation in pending and an assignee were listed as `to review` but they should be listed as `under_review`